card/lib/card/view/options.rb

```diff
@@ -51,7 +51,7 @@ module Options
           :title,          # overrides the name of the card
           :variant,        # override the canonical version of the name with a different
                            # variant
-          :editor,         # inline_nests makes a form within standard content (Symbol)
+          :input_type,     # inline_nests makes a form within standard content (Symbol)
```

Comment:
In theory this needs a migration

Reply:
Yes, but in practice I'm not sure anybody but us knows about it, and in WikiRate we only use it in code (I think).